tests/test_line_database.py

```diff
@@ -62,6 +62,14 @@ def test_line_database_from_input():
     assert ld.lines_all[0].identifier == HI.identifier
     print(ld)
 
+def test_line_database_add_line_linetools():
```

Comment:
I believe this is leftover in your stack from previous changes you've made on LineTools.  Is it possible to strip this out so we just focus on the Firefly functionality?